assets/templates/success.html.tmpl

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport"
+    content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no, shrink-to-fit=no, viewport-fit=cover, target-densityDpi=device-dpi">
+  <meta name="robots" content="noindex, nofollow, noarchive">
+  <meta property="og:title" content="{{ .PageTitle }}" />
+  <meta property="og:description" content="Successful token page." />
+  <meta name="description" content="Successful token page">
+  <meta name="apple-mobile-web-app-capable" property="apple-mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-status-bar-style" property="apple-mobile-web-app-status-bar-style"
+    media="(prefers-color-scheme: light)" content="white">
+  <meta name="apple-mobile-web-app-status-bar-style" property="apple-mobile-web-app-status-bar-style"
+    media="(prefers-color-scheme: dark)" content="black">
+  <meta name="theme-color" media="(prefers-color-scheme: light)" content="white">
+  <meta name="theme-color" media="(prefers-color-scheme: dark)" content="black">
+  <link rel="icon" type="image/png" sizes="192x192" href="/assets/static/images/favicons/android-chrome-192x192.png">
+  <link rel="icon" type="image/png" sizes="512x512" href="/assets/static/images/favicons/android-chrome-512x512.png">
+  <link rel="apple-touch-icon" sizes="180x180" href="/assets/static/images/favicons/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/assets/static/images/favicons/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="/assets/static/images/favicons/favicon-16x16.png">
+  <link rel="manifest" href="/assets/static/images/favicons/site.webmanifest">
+  <link rel="mask-icon" href="/assets/static/images/favicons/safari-pinned-tab.svg" color="#5bbad5">
+  <meta name="msapplication-TileColor" content="#da532c">
+  <meta name="theme-color" content="#ffffff">
+  <meta name="msapplication-config" content="/assets/static/images/favicons/browserconfig.xml" />
+  <meta name="msapplication-TileColor" content="#c7b3d2">
+  <meta name="msapplication-TileImage" content="mstile-144x144.png">
+  <title>{{ .PageTitle }}</title>
+</head>
+
+<body>
+</body>
+
+<script id="main" src="/assets/static/js/main.js" origin="{{ .TargetOrigin }}" token="{{ .Token }}"
```

Comment:
I have never seen this pattern of adding attributes on a script tag....need to look into this more, interesting and clever.

Reply:
I cant find anything solid to support its usage, it may have issues in various browsers. On the other hand a `data-*` prefix on custom attributes [works in html5](https://html.spec.whatwg.org/#embedding-custom-non-visible-data-with-the-data-*-attributes)

Reply:
I would suggest using `data-` prefixes instead, or just injecting these values directly.